Plugin.FirebasePushNotifications/Platforms/Android/FirebasePushNotificationManager.cs

```diff
@@ -42,7 +42,7 @@ protected override void ConfigurePlatform(FirebasePushNotificationOptions option
             notificationChannels.CreateChannels(options.Android.NotificationChannels);
 
             // TODO: Remove this code asap!!
-            this.NotificationHandler = new DefaultPushNotificationHandler();
+            //this.NotificationHandler = new DefaultPushNotificationHandler();
         }
```

Comment:
Check if anyone who still wants to use this legacy notification handler can use it.